### ISSUE TITLE
more clearly point out use of set_ and get_ to specify properties' method names

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#752]
It's not super obvious what's going on, as a client recently [pointed out to me](https://gitter.im/FakeItEasy/FakeItEasy?at=57586cde2eaa837d71e794ae):

> I saw that, but it was not clear if I need to specify 'set_', or that was what the developer chose for a name, but after you point it out, I (get) see it now

The section to update would be [Specifying a call to any method or property](http://fakeiteasy.readthedocs.io/en/stable/specifying-a-call-to-configure/#specifying-a-call-to-any-method-or-property)

**Comment by thomaslevesque** [comment]
The subtlety isn't really in FakeItEasy, but in .NET itself. Basically, you have to know that a property cannot really be "called"; only the getter and setter are methods and can be called. I suppose advanced users know that already, but it might not be obvious to everybody. I guess we could give a clearer explanation in the docs.

**Comment by blairconrad** [comment]
> The subtlety isn't really in FakeItEasy, but in .NET itself

Indeed. I did point out that I'd not heard any confusion about this, and the client was chagrined (unnecessarily so). He'd read the docs, even seeing the example with the "set_Address", but it hadn't quite clicked for the "get_". I think it was more a matter of momentary forgetfulness, rather than a lack of knowledge, but I figured there's little harm in adding another example call and maybe a couple of words in the text…
